HPXMLtoOpenStudio/resources/output.rb

```diff
@@ -294,6 +306,9 @@ def self.apply_total_loads_ems_program(model, hpxml_osm_map, hpxml_header)
       if not dehumidifier_global_vars[unit].nil?
         program.addLine("  Set loads_htg_tot = loads_htg_tot - #{dehumidifier_global_vars[unit].name}")
       end
+      defrost_load_oe_sensors[unit].each do |defrost_ss|
+        program.addLine("  Set loads_htg_tot = loads_htg_tot + #{defrost_ss.name} * #{unit_multipliers[unit]}")
```

Comment:
Try if this fixes the unit multiplier issue we had before, if CI passing I'm going to add the unit test (manually checked good).